src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -555,27 +555,22 @@
     <Compile Include="Microsoft\Data\SqlClient\PacketHandle.Unix.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SessionHandle.Unix.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true' And '$(IsPartialFacadeAssembly)' != 'true' and '$(IsUAPAssembly)' != 'true'">
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' and '$(IsUAPAssembly)' != 'true'">
     <Reference Include="Microsoft.Win32.Registry" />
   </ItemGroup>
-  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
-    <Reference Condition="'$(TargetGroup)' != 'netstandard'" Include="System.Data.Common" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
```

Comment:
We no longer need these references for netcoreapp > 2.1?

Reply:
No, they're also not needed in .NET Core 2.1, I just tested.. will remove them.
We have them all in .NET Core SDK or NuGet packages.